pkg/client/client.go

```diff
@@ -905,6 +928,25 @@ func monitorKeepTerminatedEvent(
 					return
 				}
 
+				members, err := ethereumChain.GetMembers(keepAddress)
+				memberID := tss.MemberIDFromPublicKey(operatorPublicKey)
+				memberIDs, err := tssNode.AnnounceSignerPresence(
+					ctx,
+					operatorPublicKey,
+					keepAddress,
+					members,
+				)
```

Comment:
Yet Another Refactoring Note™: I don't love that the signer presence, which is a prereq to I think all of our interactive TSS stuff, is an explicit step for the caller. It feels like BroadcastRecoveryAddress, public key generation, etc, should internally start with signer presence, and the caller just initiates the relevant protocol from the outside with a single method call. Open to feedback here though, esp from some of the folks who did the work :grimacing:

Reply:
Actually, what happened to our state machine abstraction from keep-core's gjkr beacon protocol? Is it not a good fit here? Feels like unifying these under a common state machine concept and related test infrastructure could be useful (though might not be worth it).